src/shared/components/SideNavbar/FollowedChannels.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import {
+  ChannelsWrapper,
+  ChannelsTitle,
+  ChannelsList,
+  ChannelsItem,
+  ShowMoreButton,
+  StyledChannelLink,
+} from './FollowedChannels.style'
+import { transitions } from '@/shared/theme'
+import { CSSTransition } from 'react-transition-group'
+import Icon from '../Icon'
+import { FollowedChannel } from '@/hooks/usePersonalData/localStorageClient/types'
+
+const MAX_CHANNELS = 4
+
+type FollowedChannels = {
+  followedChannels: FollowedChannel[]
+  expanded: boolean
+  onClick: () => void
+}
+
+const FollowedChannels: React.FC<FollowedChannels> = ({ followedChannels, expanded, onClick }) => {
```

Comment:
We usually add `Props` suffix to all prop types

```suggestion
type FollowedChannelsProps = {
  followedChannels: FollowedChannel[]
  expanded: boolean
  onClick: () => void
}

const FollowedChannels: React.FC<FollowedChannelsProps> = ({ followedChannels, expanded, onClick }) => {
```